pkg/util/testing/wrappers.go

```diff
@@ -550,3 +560,28 @@ func (lr *LimitRangeWrapper) WithValue(member string, t corev1.ResourceName, q s
 func (lr *LimitRangeWrapper) Obj() *corev1.LimitRange {
 	return &lr.LimitRange
 }
+
+// WorkloadPriorityClassWrapper wraps a WorkloadPriorityClass.
+type WorkloadPriorityClassWrapper struct {
+	kueue.WorkloadPriorityClass
+}
+
+// MakeWorkloadPriorityClass creates a wrapper for a WorkloadPriorityClass.
+func MakeWorkloadPriorityClass(name string) *WorkloadPriorityClassWrapper {
+	return &WorkloadPriorityClassWrapper{kueue.WorkloadPriorityClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		}},
+	}
+}
+
+// WorkloadPriorityValue update value of WorkloadPriorityClass。
```

Comment:
```suggestion
// WorkloadPriorityValue update value of WorkloadPriorityClass.
```

This is a Japanese custom :upside_down_face:

Reply:
Oh... Thanks for your review!